modules/subscriptions.php

```diff
@@ -850,7 +850,12 @@ function widget( $args, $instance ) {
 		$widget_id              = esc_attr( !empty( $args['widget_id'] )      ? esc_attr( $args['widget_id'] ) : mt_rand( 450, 550 ) );
 
 		$show_subscribers_total = (bool) $instance['show_subscribers_total'];
-		$subscribers_total      = $this->fetch_subscriber_count(); // Only used for the shortcode [total-subscribers]
+		$subscribers_total      = self::fetch_subscriber_count(); // Only used for the shortcode [total-subscribers]
```

Comment:
It may be good to update the comment here now that it will also be used for the block?